manager.go

```diff
@@ -480,7 +480,10 @@ func (m *Manager) getProgramSpec(id ProbeIdentificationPair) ([]*ebpf.ProgramSpe
 	if id.UID == "" {
 		for _, probe := range m.Probes {
 			if probe.EBPFDefinitionMatches(id) {
-				programs = append(programs, probe.programSpec)
+				// If Probe is excluded the program spec will be nil
+				if probe.programSpec != nil {
```

Comment:
This is OK for now, but I think we should probably upon `Init`, remove excluded probes from the list or return an error.